chunky/src/java/se/llbit/chunky/renderer/scene/PathTracer.java

```diff
@@ -459,6 +441,71 @@ public static boolean pathTrace(Scene scene, Ray ray, WorkerState state, int add
     return hit;
   }
 
+  private static void translucentRayColor(Scene scene, Ray ray, Ray next, double opacity) {
+    // Color-based transmission value
+    double colorTrans = (ray.color.x + ray.color.y + ray.color.z)/3;
+    // Total amount of light we want to transmit (overall transparency of texture)
+    double shouldTrans = 1 - opacity;
+    // Amount of each color to transmit - default to overall transparency if RGB values add to 0 (e.g. regular glass)
+    double rTrans = shouldTrans, gTrans = shouldTrans, bTrans = shouldTrans;
```

Comment:
Would the code get better with `trans = new Vector3(shouldTrans, shouldTrans, shouldTrans)` ?

Reply:
I just did that, was able to eliminate a few repeated statements